public/static/docs/command-reference/get.md

```diff
@@ -58,11 +58,10 @@ name.
   an existing directory is specified, then the output will be placed inside of
   it.
 
-- `--rev` - specific
-  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
-  (such as a branch name, a tag, or a commit hash) of the repository to download
-  the file or directory from. The tip of the default branch is used by default
-  when this option is not specified.
+- `--rev` - specific [Git revision](https://git-scm.com/docs/revisions) (such as
+  a commit SHA hash, or a branch or tag name) of the repository to download the
```

Comment:
same here - start with `commit hash, brach or tag name (specific Git revision...) of the repo to download data from`

Reply:
Good idea. Updated in c006d18a.